### ISSUE TITLE
The Boundless Stabba

### ISSUE THREAD

**Issue by mursie** [WarEmu/WarBugs#4460]
Squig herder spear stats.  Currently has 40 strength.

can this be converted to ballistic please.

**Comment by DanaBloodkiss** [comment]
No :)

**Comment by mursie** [comment]
Please re-consider this.  All sc weapons for squig are ballistic based.  The re-designed genisaurus suits are ballistic based... when you switch to squig armor your ballistic is now converted to strength etc...   This stat layout is from an archaic time when at game first launch when strength melee squigs were still thought to maybe be a thing.  Mythic quickly reversed that course but this archaic stat layout apparently hasn't been touched.

Simply put - strength shouldn't be on the squig herder weapon.  Your dismissal is counter to all weapons made by mythic for the squig prior to close and even the sc's weapons you currently have in game for squig.
